apiserver/src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -166,23 +244,194 @@ public void getProjectsWithAclEnabledTest() {
+        assertThatJson(getPlainTextBody(response)).isEqualTo("""
+                [
+                  {
+                      "name" : "acme-app-b",
+                      "uuid" : "${json-unit.any-string}",
+                      "tags" : [ {
+                        "name" : "foo"
+                      } ],
+                      "externalReferences":[],
+                      "isLatest" : false,
+                      "active" : true
+                  }
+                ]
+                """);
+    }
+
+    @Test
+    public void getProjectsNotAssignedToTeamWithUuidTest() {
+        final var projectA = new Project();
+        projectA.setName("acme-app-a");
+        qm.persist(projectA);
+
+        final var projectB = new Project();
+        projectB.setName("acme-app-b");
+        qm.persist(projectB);
+        projectB.setAccessTeams(Set.of(team));
+
+        // Should exclude projectB as it is assigned to the team.
+        Response response = jersey.target(V1_PROJECT)
+                .queryParam("notAssignedToTeamWithUuid", team.getUuid())
+                .request()
+                .header(X_API_KEY, apiKey)
+                .get();
+        assertThat(response.getStatus()).isEqualTo(200);
+        assertThat(response.getHeaderString(TOTAL_COUNT_HEADER)).isEqualTo("1");
+        assertThatJson(getPlainTextBody(response)).isEqualTo("""
+                [ {
+                   "name" : "acme-app-a",
+                   "uuid" : "${json-unit.any-string}",
+                   "externalReferences" : [ ],
+                   "isLatest" : false,
+                   "active" : true
+                 } ]
+                """);
+    }
+
     @Test
     public void getProjectsByNameRequestTest() {
-        for (int i = 0; i < 1000; i++) {
-            qm.createProject("Acme Example", null, String.valueOf(i), null, null, null, null, false);
+        for (int i = 0; i < 10; i++) {
+            qm.createProject("Acme Example " + i, null, String.valueOf(i), null, null, null, null, false);
         }
         Response response = jersey.target(V1_PROJECT)
-                .queryParam("name", "Acme Example")
+                .queryParam("name", "Acme Example 7")
                 .request()
                 .header(X_API_KEY, apiKey)
                 .get(Response.class);
         Assert.assertEquals(200, response.getStatus(), 0);
-        Assert.assertEquals(String.valueOf(1000), response.getHeaderString(TOTAL_COUNT_HEADER));
+        Assert.assertEquals(String.valueOf(1), response.getHeaderString(TOTAL_COUNT_HEADER));
         JsonArray json = parseJsonArray(response);
         Assert.assertNotNull(json);
-        Assert.assertEquals(100, json.size());
+        Assert.assertEquals(1, json.size());
+        Assert.assertEquals("Acme Example 7", json.getJsonObject(0).getString("name"));
+        Assert.assertEquals("7", json.getJsonObject(0).getString("version"));
+    }
```

Comment:
I feel like the previous test should continue to work as-is. The new test of only retrieving a single project here should be separate.